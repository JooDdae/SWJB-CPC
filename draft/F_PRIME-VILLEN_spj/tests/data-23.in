8537869339237823915137963539431325312232687823115135573533982938337879541336176199850176494999154390492423412771592689900122816089467535158695227943114396577947311879007979134993821941392367815669736932718297880385734019557312497579491859544372536659130757791103593994678231462193711013373913276857169333182876521632340032371866991279323349269923393343316757179076863782993976917391993491049
