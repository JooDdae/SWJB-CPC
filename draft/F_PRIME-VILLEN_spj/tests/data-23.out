184
147 4 1083 544 4 143 1134 2 1006 3 547 0 2 1 3 2 0 1 1 2 390 143 0 0 3 0 2 3 3 4 2 3 2 2 1212 2 23 2 996 3 13 2 2 18 4 806 1060 971 669 0 3 14 1125 360 13 1 4 4 0 17 391 1118 1 60 794 91 3 2 3 0 773 3 1 1 22 14 0 0 85 851 161 2 195 1119 1208 2 95 2 1030 570 1 2 873 747 938 2 1 4 0 145 4 1096 148 2 555 3 3 4 2 204 4 3 1004 0 149 3 86 4 1 3 2 858 0 63 3 4 22 0 27 780 1173 143 0 624 159 0 26 2 2 131 0 2 1 4 882 264 2 2 0 1039 843 823 552 1 2 4 0 1079 1131 1154 70 393 346 2 2 14 2 0 19 3 4 0 4 155 883 4 145 1162 125 4 2 1140 70 176 